### PR TITLE
[webidl-conversions] Upgrade to v7

### PR DESCRIPTION
**Pull request by BendingBender**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jsdom/webidl-conversions
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/webidl-conversions/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
-// Type definitions for webidl-conversions 6.1
+// Type definitions for webidl-conversions 7.0
 // Project: https://github.com/jsdom/webidl-conversions#readme
 // Definitions by: ExE Boss <https://github.com/ExE-Boss>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
```
Comment (ExE-Boss) [inline types/webidl-conversions/index.d.ts:6]: This line is no longer necessary, as the minimum supported **TypeScript** version by @DefinitelyTyped is greater than **3.0**.
```suggestion
```

**Review by ExE-Boss — COMMENTED** [review]
The parameter types of `any` can be changed to `unknown`.

File: types/webidl-conversions/index.d.ts (hunk 1 above)
Comment (BendingBender) [reply types/webidl-conversions/index.d.ts]: done

File: types/webidl-conversions/index.d.ts [hunk 2]
```diff
@@ -1,17 +1,15 @@
-// Type definitions for webidl-conversions 6.1
+// Type definitions for webidl-conversions 7.0
 // Project: https://github.com/jsdom/webidl-conversions#readme
 // Definitions by: ExE Boss <https://github.com/ExE-Boss>
+//                 BendingBender <https://github.com/BendingBender>
```
Comment (jakebailey) [inline types/webidl-conversions/index.d.ts:4]: Same question as your other PR; are you looking to be come an owner/reviewer, or just authorship credit? https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Comment (BendingBender) [reply types/webidl-conversions/index.d.ts:4]: Yes, that's what I wanted.

**Comment by sandersn** [comment]
@ExE-Boss @BendingBender addressed your comments. Do you want to take another look and sign off? If not, I'll probably merge in a couple of days.

**Pull request merged**